lib/tags/input-text.tag

```diff
@@ -176,7 +176,7 @@
         }
 
         this.sanitizeHTML = (event) => {
-            event.preventDefault();
+            //event.preventDefault();
```

Comment:
maybe this is what's causing?
<img width="633" alt="Screen Shot 2019-05-16 at 8 31 35 PM" src="https://user-images.githubusercontent.com/19166/57874683-d08e1a00-7819-11e9-9a1f-c85682a40a2d.png">